.github/E2E_TESTS_ON_GITHUB_CI.md

```diff
@@ -1,58 +1,87 @@
+Most workflows may be triggered with default configs. Some, nevertheless, may be overridden.
 
-Config overrides should be stored in `testconfig/*/overrides/*.toml`. Placing files here will not trigger a rebuild of the test runner image.
+> [!TIP]
+> Use `gh` CLI commands to run workflows from local machine.
 
-**Important Note:** The use of `base64Config` input is deprecated in favor of `test_config_override_path`. For more details, refer to [the decision log](https://smartcontract-it.atlassian.net/wiki/spaces/TT/pages/927596563/Storing+All+Test+Configs+In+Git).
+#### Configuration Overrides
 
-To learn more about test configs see [CTF Test Config](https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/lib/config/README.md).
+1. Reference sources:
+   1. [Integration-Tests configurations](../integration-tests/testconfig/README.md);
+   2. [CTF Test Config](https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/lib/config/README.md).
+2. Defaults and overrides should be stored (committed) in repository under `../integration-tests/testconfig/<product>/overrides/<override-name>.toml` (see example [here](../integration-tests/testconfig/ocr2/overrides/base_sepolia.toml)).
+3. Use `test_config_override_path` to point to an override config. For example: `test_config_override_path="testconfig/ocr2/overrides/base_sepolia.toml"`
 
-## Test Secrets
+#### Test Secrets
 
-For security reasons, test secrets and sensitive information are not stored directly within the test config TOML files. Instead, these secrets are securely injected into tests using environment variables. For a detailed explanation on managing test secrets, refer to our [Test Secrets documentation](https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/lib/config/README.md#test-secrets).
+> [!RISK]
+> Pay attention to never store/expose/commit your test secrets in repository.
 
-If you need to run a GitHub workflow using custom secrets, please refer to the [guide on running GitHub workflows with your test secrets](https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/lib/config/README.md#run-github-workflow-with-your-test-secrets).
+Test secrets allow provisioning and override the sensitive data such as EOA's private key, RPCs, Docker registry links, etc.
 
-## About the E2E Test Reusable Workflow
+Reference sources:
 
-For information on the E2E Test Reusable Workflow, visit the documentation in the [smartcontractkit/.github repository](https://github.com/smartcontractkit/.github/blob/main/.github/workflows/README.md).
+1. [BASE64_CONFIG_OVERRIDE](../integration-tests/testconfig/README.md#base64_config_override).
```

Comment:
@chudilka1 I'd explicitly say that **test configs** (link to docs in CTF) are used for non-secrets and **test secrets** for secrets (link to docs in CTF)

Reply:
I've added the risk sections to address it. Though, IMHO, there is no need, as there is a section about [test/node configuration](https://github.com/smartcontractkit/chainlink/pull/14577/files#diff-c8e584763fd2bd4e5ef372d8f5221ed841afbb22920e64af239148786d6caa69R61).